### ISSUE TITLE
False postive Buttons must have discernible text

### ISSUE THREAD

**Issue by yonjah** [dequelabs/axe-core#1652]
Expectation: 
I expect both of this items to be considered as identical
```html
<button id="btn1" title="Delete">
	<span class="fa fa-trash"></span>
</button>

<button id="btn2" title="Delete"><span class="fa fa-trash"></span></button>
```

Actual: 
`btn1` will cause a violation  `Buttons must have discernible text`
`btn2` will pass without violations

Motivation: 
Not sure if this is a bug or expected behavior from the spec.
Checking the rule code it seem that halting on the first whitespace only value and ignoring the rest is intentional https://github.com/dequelabs/axe-core/blob/a84431ab3c37eccf43c3065506af3ad8097e981d/lib/commons/text/accessible-text-virtual.js#L53

<pre><code>
axe-core version: 3.2.2
axe-webdriver.js version: 2.2.0
</code></pre>

**Comment by WilcoFiers** [comment]
Thank you for reporting. Yeah that is odd. We'll have to look into this.

**Comment by straker** [comment]
These seems to be fixed in the latest `develop`. It does produces failures in v3.2.2, but the latest axe-coconut is not reporting the issue.

**Comment by yonjah** [comment]
thanks @straker
I checked with axe-coconut now and it seem to work ok.
So this should eventually be available in the webdriver as well ?

**Comment by straker** [comment]
Yep. We have a release coming up for axe v3.3.0. Once that's released, we'll go around to the different integrations and update the version. Hopefully that should happen within a week or two after the release.

**Comment by straker** [comment]
Just so you're aware, we released axe v3.3.0 yesterday and will soon be updating the integrations. I'll let you know when webdriver is updated with the latest version.

**Comment by straker** [comment]
axe-webdriverjs v2.3.0 was just released and hopefully should solve this problem.
